packs/redis/templates/redis.conf.tpl

```diff
@@ -0,0 +1,2087 @@
+# Redis configuration file example.
```

Comment:
I think with this we'll want to make this file a helper like those in _helpers

And then optionally include it in a template stanza and pass it into the docker command (assuming something like "use redis.conf file" is true)

Reply:
Er... on second thought this is very long 🤔 so making a helper template from it seems wrong.